File: JunsBlog/Models/Authentication/UserInfoUpdateRequest.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace JunsBlog.Models.Authentication
{
    public class UserInfoUpdateRequest
    {
        public string Id { get; set; }
        public string Name { get; set; }
        public string Email { get; set; }
        public string Image { get; set; }
    }
}

File: JunsBlog/ClientApp/src/app/models/userInfoUpdateRequest.ts
export class UserInfoUpdateRequest {
    image: string;
    name: string;
    id: string;
    email: string;
    constructor(id: string, name: string, email: string, image: string){
        this.id = id;
        this.name = name;
        this.email = email;
        this.image = image;
    }
}
